packages/opencensus-core/src/common/validations.ts

```diff
@@ -43,3 +45,27 @@ export function validateArrayElementsNotNull<T>(
     throw new Error(`${errorMessage} elements should not be a NULL`);
   }
 }
+
+/** Throws an error if any of the map elements is null. */
+export function validateMapElementNotNull<T>(
+    map: Map<T, T>, errorMessage: string) {
+  for (const [key, value] of map.entries()) {
+    if (key === null || key === undefined || value === null ||
```

Comment:
Optional: you can use `== null` to test for both null and undefined, which is a permissible exception IMO to general (good) advice to always use `===`. See https://stackoverflow.com/questions/2559318/how-to-check-for-an-undefined-or-null-variable-in-javascript for details.

Reply:
SGTM, done. Thanks for sharing the link.